doc/source/ray-overview/getting-started.md

```diff
@@ -672,10 +660,10 @@ Ray dashboard provides a visual interface that displays real-time system metrics
 ```
 
 ````{note}
-To get started with ray dashboard install the Ray default installation as follows.
+To get started with the dashboard, install the default installation as follows:
 
 ```bash
-pip install "ray[default]"
+pip install -U "ray[default]"
 ```
 ````
```

Comment:
Should there be more steps after installation? 😅

Reply:
Added a step for where to find the dashboard.